meta-ostro-xt/recipes-opencl/ocl-icd/ocl-icd/configurable-doc-generation.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/configure.ac b/configure.ac
```

Comment:
@kad, Thanks for pointing out, I was in impression that git formatted patch fail to apply on non-git source, now I git formatted the patch.